### PR TITLE
Update Cartridge to 2.3.0 in tests

### LINKED ISSUE #115: [1pt] Fix edit user idempotence check for cartridge 2.3.0
In a new versions of cartridge, user has `version` field that is changed on every user updte. This field should be ignoring on comparing new user with the old one.

### PR DESCRIPTION
**Pull request by dokshina**
Closes #115

### CODE REVIEW

File: library/cartridge_auth.py [hunk 1]
```diff
@@ -157,6 +157,27 @@ def user_is_deleted(user):
     return 'deleted' in user and user['deleted'] is True
 
 
+def users_are_equal(user1, user2):
+    for key in user1:
+        if key == 'version':
```
Comment (olegrok) [inline library/cartridge_auth.py:162]: Why not smth similar to
```py
user1_cpy = user1.copy()
user2_cpy = user2.copy()

# Maybe additional check needed here
del user1_cpy['version']
del user2_cpy['version']

return user1_cpy == user2_cpy
```

File: unit/test_auth.py [hunk 2]
```diff
@@ -249,6 +249,65 @@ def test_edit_user(self):
...
+
+        USER = {
+            'username': 'dokshina',
+            'email': 'dokshina@tarantool.love',
+            'fullname': 'Elizaveta Dokshina',
+        }
+
+        new_params = {
+            'password': 'new-password',
+            'email': 'dokshina@tarantool.hate',
```
Comment (olegrok) [inline unit/test_auth.py]:
```suggestion
            'email': 'dokshina@tarantool.love_a_bit_more',
```

File: library/cartridge_auth.py (hunk 1 above)
Comment (dokshina) [reply library/cartridge_auth.py:162]: What's the difference?

Comment (olegrok) [reply library/cartridge_auth.py:162]: My suggestion is shorter

**Review by olegrok — APPROVED** [review]

**Pull request merged**